tests/bearlib/aspects/ClassTest.py

```diff
@@ -56,3 +57,21 @@ class SubAspect:
         assert (str(exc.value) == "<class 'str'> is not an aspectclass or "
                 'an instance of an aspectclass')
         assert issubaspect(SubAspect('Python'), RootAspect)
+
+
+class AspectlistClassTest:
+
+    def test__init__(self):
+        with pytest.raises(TypeError) as exc:
+            aspectlist(['String'])
+        assert (str(exc.value) == "'String' is not an aspectclass or "
+                'an instance of an aspectclass')
+
+    def test__contains__(self):
+        list_of_aspect = aspectlist(
+            [Metadata.CommitMessage.Shortlog, Metadata.CommitMessage.Body])
```

Comment:
IMO it's pretty weird to use real aspects in test. If something happen to those aspect (change of structure, moving, etc), then it potentially break unrelated test. This also include https://github.com/adhikasp/coala/blob/ef82c22d60da32298b486d49c792bba46036fcc8/tests/bears/BearTest.py#L112

Maybe we need to make a special aspect tree for test purpose? 
Ex. `TestRoot`, `TestRoot.Sub1`, `TestRoot.Sub1.Sub1`, `TestRoot.Sub2`, and so on.